recipes/libyuv/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibyuvConan(ConanFile):
+    name = "libyuv"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://chromium.googlesource.com/libyuv/libyuv/"
+    description = "libyuv is an open source project that includes YUV scaling and conversion functionality."
+    topics = ["YUV", "libyuv", "google", "chromium"]
+    license = "BSD-3-Clause"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "with_jpeg": [False, "libjpeg", "libjpeg-turbo"]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "with_jpeg": "libjpeg"}
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        if self.options.with_jpeg == "libjpeg":
+            self.requires("libjpeg/9d")
+        elif self.options.with_jpeg == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/2.0.5")
```

Comment:
libjpeg-turbo is an invalid option right now, as it is not supported by the cmake script.
It might get added in the future. So maybe throw a Conan invalid configuration?

Reply:
I try `conan create . libyuv/cci.20201106@_/_ -o libyuv:with_jpeg=libjpeg-turbo` and it is OK. Why libjpeg-turbo is an invalid option right now?

Reply:
Does it really use the (static) conan libjpeg-turbo libraries?
https://github.com/conan-io/conan-center-index/blob/f348bd57ae084d9b7fd43c3c136733023bc2807c/recipes/libjpeg-turbo/all/conanfile.py#L141

Reply:
I do a test. My system (Manjaro Linux) does not has any libjpeg.a in `/usr/lib` and I build pass(`conan create . libyuv/cci.20201106@_/_ -o libyuv:with_jpeg=libjpeg-turbo -o libyuv:shared=True -o libjpeg-turbo:shared=False`. `ldd` show no libjpeg.so is needed which means it link a static libjpeg.

Reply:
Yes, I will fix this.

Reply:
Fixed